data-requirements/README.md

```diff
@@ -0,0 +1,54 @@
+# fqm-execution Data Requirements Output Testing/Comparison
```

Comment:
Is it worth adding a link to this from the main README?

Reply:
Honestly, I am not sure. If you think it will be helpful, then definitely! I do notice that we don't mention anything about the regression directory and tests in the main README although I am not sure if that was purposeful or also something we want to add to the README.